addOns/wappalyzer/src/main/java/org/zaproxy/zap/extension/wappalyzer/WappalyzerPassiveScanner.java

```diff
@@ -166,29 +170,33 @@ private void checkMetaElementsMatches(Source source) {
         }
     }
 
-    private void checkDomElementMatches(Source source) {
-        Document doc = Jsoup.parse(String.valueOf(source));
+    private void checkDomElementMatches(HttpMessage msg, Source source) {
```

Comment:
Source is no longer necessary here, right?

Reply:
no, it is not required.

Reply:
I have made changes in the previous commit itself.